content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -59,13 +59,16 @@ Use frontend data from RUM, as well as backend, infrastructure, and log informat
         clientToken: '<DATADOG_CLIENT_TOKEN>',
         ...otherConfig,
         service: "my-web-application",
-        allowedTracingOrigins: ["https://api.example.com", /https:\/\/.*\.my-api-domain\.com/, (origin) => origin === "https://api.example.com"]
+        allowedTracingUrls: ["https://api.example.com", /https:\/\/.*\.my-api-domain\.com/, (origin) => origin === "https://api.example.com"]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏not origin anymore
```suggestion
        allowedTracingUrls: ["https://api.example.com", /https:\/\/.*\.my-api-domain\.com/, (url) => url.startsWith("https://api.example.com")]
```